firefox-ios/Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -325,6 +326,19 @@ class LibraryViewController: UIViewController, Themeable {
         navigationController?.toolbar.tintColor = theme.colors.actionPrimary
     }
 
+    private func updateSegmentControl() {
+        if isBookmarkRefactorEnabled {
+            let panelState = getCurrentPanelState()
+
+            switch panelState {
+            case .bookmarks(state: .inFolderEditMode):
+                (1...3).forEach { index in self.librarySegmentControl.setEnabled(false, forSegmentAt: index) }
+            default:
+                (0...3).forEach { index in self.librarySegmentControl.setEnabled(true, forSegmentAt: index) }
```

Comment:
Can we avoid magic numbers? I am not sure why it's `1...3` and `0...3`. Having some constants with a clear name would make it easier to understand I think